### DISCUSSION TITLE
Request to raise PR 366 from the dead

### DISCUSSION THREAD

**Discussion by ulisse31** [JohanDegraeve/xdripswift#474] [General]
Hi all, 
This PR #366 Implemented Insulin on Board is in my opinion a killer feature, _extremely_ helpful in blood glucose management.
This feature is built-in in the Android version, and implementing it in IOS would be a great enhancement and another step towards feature parity.

In the PR discussion I see good advices but no strong pushbacks, so in a real agile fashion we could implement this feature as-is and then iterate further improvements later on?

What would be the correct step(s) to resume and deploy?

thank you for your work
Andrea

**Comment by paulplant** [comment]
Sounds like a good idea!
